build/cloud/05-configmap.yaml

```diff
@@ -26,6 +26,14 @@ data:
       keepalive-interval: 30
       write-timeout: 30
       node-limit: 10
+    devicecontroller:
+       kube:
+         master: https://kubernetes.default.svc.cluster.local:443
+         namespace: "kubeedge"
```

Comment:
The controller should have namespace as empty string (""). 
For kubeclient "" means AllNamespaces